doc/source/gettingstarted.rst

```diff
@@ -68,11 +79,13 @@ Installing from cloned source code (recommended for developers)
 
 
 Troubleshooting:
- - *Optional* Set the ``PYTHONPATH`` environment variable to the location of your fast-trips repo, which we're calling ``<fast-trips-dir>``.
+ - *Optional* Set the ``PYTHONPATH`` environment variable to the location of your fast-trips repo, which we're calling
+   ``<fast-trips-dir>``.
 
 .. note::
  - Pandas 0.21.x has known issues, and it is not compatible with Fast-Trips.
```

Comment:
Let's remove this line? The requirements should be >=1.0 or something

Reply:
Great catch. Fixed